server/index.js

```diff
@@ -1,46 +1,67 @@
-const express = require('express');
-const path = require('path');
+const express = require("express");
+const app = express();
+const http = require("http");
+const cors = require("cors");
+const path = require("path");
+const { Server } = require("socket.io");
 
-const PORT = 3000;
+const PORT = 8080;
 
-const app = express();
+app.use(cors());
 app.use(express.json());
+app.use(express.static(path.join(__dirname, "../client/dist")));
 
-app.use(express.static(path.join(__dirname, '../client/dist')));
-
+const server = http.createServer(app);
 
-app.get('/Auth', (req, res) => {
-  console.log(`Request at ${`/Auth`}`)
-  res.status(200).send({ AuthData: true })
-})
+const io = new Server(server, {
+  cors: {
+    origin: "http://localhost:3000",
+    methods: ["GET", "POST"]
+  }
+});
 
-app.get('/Browse', (req, res) => {
-  console.log(`Request at ${`/Browse`}`)
-  res.status(200).send({ BrowseData: true })
-})
+io.on("connection", (socket) => {
+  console.log(`User connected 😀: ${socket.id}`);
+  socket.on("send_message", (data) => {
```

Comment:
I like the emoji, might be safer to use its unicode code if possible. If you don't have time to implement this, don't worry about it!